WalletWasabi.Gui/Global.cs

```diff
@@ -754,7 +754,7 @@ public async Task DisposeAsync()
 
 				try
 				{
-					SingleInstanceChecker?.Dispose();
+					await SingleInstanceChecker.DisposeAsync().ConfigureAwait(false);
```

Comment:
Add back nullcheck.

Reply:
Don't need it, SingleInstanceChecker created with ctor.